### ISSUE TITLE
'Could not create directory'

### ISSUE THREAD

**Issue by raboof** [akka/akka-http#2456]
https://jenkins.akka.io:8498/job/akka-http-nightly/AKKA_VERSION=default,Node=akka-http-nightly-node,SCALA_VERSION=2.13.0-M5,jdk=AdoptOpenJDK%208/413/console

```
[error] java.lang.RuntimeException: Could not create directory /home/jenkins/workspace/workspace/akka-http-nightly/AKKA_VERSION/default/Node/akka-http-nightly-node/SCALA_VERSION/2.13.0-M5/jdk/AdoptOpenJDK 8/akka-http-testkit/target/scala-2.13.0-M5/classes.bak
[error] 	at scala.sys.package$.error(package.scala:26)
[error] 	at sbt.io.IO$.createDirectory(IO.scala:326)
[error] 	at sbt.internal.inc.ClassFileManager$TransactionalClassFileManager.<init>(ClassFileManager.scala:73)
[error] 	at sbt.internal.inc.ClassFileManager$.transactional(ClassFileManager.scala:67)
[error] 	at sbt.internal.inc.ClassFileManager$.getDefaultClassFileManager(ClassFileManager.scala:32)
[error] 	at sbt.internal.inc.ClassFileManager$.getClassFileManager(ClassFileManager.scala:39)
[error] 	at sbt.internal.inc.Incremental$.manageClassfiles(Incremental.scala:133)
[error] 	at sbt.internal.inc.Incremental$.compile(Incremental.scala:80)
[error] 	at sbt.internal.inc.IncrementalCompile$.apply(Compile.scala:67)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:311)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.$anonfun$compileIncrementally$1(IncrementalCompilerImpl.scala:269)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.handleCompilationError(IncrementalCompilerImpl.scala:159)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compileIncrementally(IncrementalCompilerImpl.scala:238)
[error] 	at sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:69)
[error] 	at sbt.Defaults$.compileIncrementalTaskImpl(Defaults.scala:1549)
[error] 	at sbt.Defaults$.$anonfun$compileIncrementalTask$1(Defaults.scala:1523)
```

**Comment by jrudolph** [comment]
Seems related: https://github.com/sbt/sbt/issues/1673

**Comment by jrudolph** [comment]
Didn't happen for a long time and something seems to have been fixed in sbt 1.3.3.
